lms/djangoapps/ccx/tests/test_field_override_performance.py

```diff
@@ -242,7 +242,7 @@ class TestFieldOverrideMongoPerformance(FieldOverridePerformanceTestCase):
     __test__ = True
 
     # TODO: decrease query count as part of REVO-28
-    QUERY_COUNT = 31
+    QUERY_COUNT = 37
```

Comment:
I'm a little surprised the query count went up so much. Isn't it request cached?

Reply:
The new model's `override_value()` method *is* cached:
https://github.com/openedx/edx-platform/blob/b61ea3d67c5d7ea9e0fe61c020e24b940b242f5c/openedx/core/djangoapps/waffle_utils/models.py#L103
I'll look more deeply into a few query count mismatches to see what's causing the increased counts.

Reply:
@ormsbee In this particular test, all waffle flag queries were being ignored in query counts - but I hadn't added the new query to that list. Here's [that list](https://github.com/openedx/edx-platform/blob/3bff73c073a2502356a5cebe5f8cd6de699ce313/openedx/core/djangoapps/waffle_utils/testutils.py#L9-L15). There are 6 unique waffle flag checks that are happening in this test, which is why the query count increased by 6 queries before I added the new query to the `WAFFLE_TABLES` ignore list.

tl;dr - The query count increased only by the expected amount.